extensions/wikia/ApiDocs/services/CachingApiDocsService.php

```diff
@@ -20,7 +20,7 @@ class CachingApiDocsService implements IApiDocsService {
 	private $cacheTime = 43200; // 60 * 60 * 12
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app